### DISCUSSION TITLE
download/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#609] [Announcements]
# download/

资源下载 点击下载样例开发包 (http://39.152.31.83:11280/apps/epsplanet_docs/resource/demo.zip) ::: tip 开发模板，已经配置完毕，可以直接进行开发。 ::: 点击下载 EPSPlanetSDK (http://39.152.31.83:11280/apps/sunwayDownlo...

http://127.0.0.1:8080/download/

**Comment by AagamShah2314** [comment]
Where is download link?

**Reply by AbandonedCart** [reply]
The buttons above the changelog
